src/components/Toast/ToastStyle.ts

```diff
@@ -0,0 +1,44 @@
+import {StyleSheet} from 'react-native';
+
+
+const styles = StyleSheet.create({
+    successToastContainer: {
+        backgroundColor: '#def1d7',
+        borderColor: '#1f8722',
+    },
+    warningToastContainer: {
+        backgroundColor: '#fef7ec',
+        borderColor: '#f08135',
+    },
+    errorToastContainer: {
+        backgroundColor: '#fae1db',
+        borderColor: '#d9100a',
+    },
+    successToastText: {
+        color: '#1f8722',
+    },
+    warningToastText: {
+        color: '#f08135',
+    },
+    errorToastText: {
+        color: '#d9100a',
+    },
+})
+
+export const toastStyles = {
```

Comment:
We want to use it in indexed manner in `src/components/Toast/Toast.tsx`. In this case type definitions are required